estimate_gain: 1.5